src/app/style.module.scss

```diff
@@ -0,0 +1,21 @@
+.Header{
+    background-color: white !important ;
+    color: black !important ;
+    .MenuIcon{
+        background-color: aqua !important;
+    }
+
+}
+
+.SideBar{
+    display: flex;
+    flex-direction: column;
+    margin: 0 5px;
+    height: 100vh;
+}
+
```

Comment:
where you use all this classes , i didn't see all except container??

Reply:
and also css classes naming begin with small letter not capital , take care of that.